### ISSUE TITLE
In Dev subsiteCategory is multiplied

### ISSUE THREAD

**Issue by mikezimm** [mikezimm/pivottiles7#18]
## In Dev (clicky) subsiteCategory is multiplied

![image](https://user-images.githubusercontent.com/49648086/100816082-5af59900-3413-11eb-8159-8b29b871fbc3.png)

**Comment by mikezimm** [comment]
## Updated buildTileCategoriesFromResponse
1. buildTileCategoriesFromResponse only adds now if that category is not already in the list.
2. This is a bandaide and works and it's not worth figuring out the complex logic about what is causing it.

```typescript
  if ( hasSubsites === true && tileCategories.indexOf(pivotProps.fetchInfo.subsitesCategory) < 0 ) {
    tileCategories.push( pivotProps.fetchInfo.subsitesCategory );
  }

  if ( hasHubs === true && tileCategories.indexOf(pivotProps.fetchInfo.hubsCategory) < 0 ) {
    tileCategories.push( pivotProps.fetchInfo.hubsCategory );
  }

  if ( hasLists === true && tileCategories.indexOf(pivotProps.fetchInfo.listCategory) < 0 ) {
    tileCategories.push( pivotProps.fetchInfo.listCategory);
  }

  if ( hasLibraries === true && tileCategories.indexOf(pivotProps.fetchInfo.libsCategory) < 0 ) {
    tileCategories.push( pivotProps.fetchInfo.libsCategory );
  }
```

**Comment by mikezimm** [comment]
Fixed with #b3c6638
